apps/api/src/domain/models/user-management/group/commands/group-created.event.ts

```diff
@@ -0,0 +1,11 @@
+import { AggregateId } from '../../../../types/AggregateId';
+import { BaseEvent } from '../../../shared/events/base-event.entity';
+import { CreateGroup } from './create-group.command';
+
+export class GroupCreated extends BaseEvent {
+    type = 'USER_GROUP_CREATED';
+
+    constructor(command: CreateGroup, eventId: AggregateId) {
```

Comment:
Does it make any difference if we skip overriding the constructor on the child class here? That would be super simple!

Reply:
it worked. done.